full/Dockerfile

```diff
@@ -147,9 +147,9 @@ RUN curl -fsSL https://github.com/pyenv/pyenv-installer/raw/master/bin/pyenv-ins
     && pip3 install --upgrade pip \
     && pip3 install virtualenv pipenv pylint rope flake8 autopep8 pep8 pylama pydocstyle bandit python-language-server[all]==0.25.0 \
     && rm -rf /tmp/*
-# use global installation location for pip modules
-ENV PYTHONUSERBASE=/workspace/.pip-modules \
-    PIP_USER=yes
+# Gitpod will automatically add user site under `/workspace` to persist your packages.
+# ENV PYTHONUSERBASE=/workspace/.pip-modules \
+#    PIP_USER=yes
```

Comment:
Nit: Why are you leaving the code commented-out here? As a sort of documentation/warning to users who'd be tempted to set `PYTHONUSERBASE` to something else?

Reply:
yes, that was the idea. would you like to have this removed, but leaving the comment line?

Reply:
I don't really mind either way. I think this may be overly cautious, but it doesn't hurt to have an extra comment in the Dockerfile, so fine either way. 😄 👍